cfme/fixtures/networks.py

```diff
@@ -29,18 +29,20 @@ def test_something(targeted_refresh)
                     targeted_refresh.register_target('ref-aaaabb', 'Router named TEST')
     """
     evm_log = '/var/www/miq/vmdb/log/evm.log'
```

Comment:
I think we no longer need this in separate variable as we only use it once. Please consider passing it directly.